src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -32,59 +27,20 @@ export const PartyMap: React.FC = () => {
   const { recentVenueUsers } = useRecentVenueUsers();
 
   const currentVenue = useSelector(partyMapVenueSelector);
-  const venues = useSelector(orderedVenuesSelector);
 
-  const [isRoomModalOpen, setRoomModalOpen] = useState(false);
   const [selectedRoom, setSelectedRoom] = useState<Room | undefined>();
 
   const selectRoom = useCallback((room: Room) => {
     setSelectedRoom(room);
-    setRoomModalOpen(true);
   }, []);
 
-  // Note: we explicitly don't unset selectedRoom here as that would cause RoomModal to close abruptly
   const unselectRoom = useCallback(() => {
-    setRoomModalOpen(false);
+    setSelectedRoom(undefined);
   }, []);
```

Comment:
Why was this changed? It's a regression and should be reverted. (see other comments in this review for further explanation)

Reply:
I don't think we need `isRoomModalOpen` entity since it is a dupe of `selectRoom`

Reply:
It's not a dupe (as explained elsewhere in the comments, though let me know if you still don't understand the nuance of why). We need to be able to hide/unhide the modal without unsetting the venue it has, to avoid 'unrendering' it, and to allow our 'smooth css transitions' to run.